src/python/zquantum/core/utils_test.py

```diff
@@ -304,19 +308,10 @@ def test_two_value_estimates_are_equal_iff_their_values_and_precisions_are_equal
 
 
 @pytest.mark.parametrize(
-    "estimate",
-    [
-        ValueEstimate(2.0),
-        ValueEstimate(5.0, precision=1e5)
-    ]
+    "estimate", [ValueEstimate(2.0), ValueEstimate(5.0, precision=1e5)]
```

Comment:
It doesn't seem to really matter, but precision here is `1e5`, shouldn't it be `1e-5`?

Reply:
ccf2ad5